controller/SqlCommands.cs

```diff
@@ -0,0 +1,130 @@
+﻿using MySqlConnector;
+using ZooAPI.model;
+
+namespace ZooAPI.controller;
+
+public class SqlCommands
+{
+    static MySqlConnectionStringBuilder builder = new MySqlConnectionStringBuilder
+    {
+        Server = "localhost",
+        Database = "Zoo",
+        UserID = "root",
+        Password = "admin",
+        SslMode = MySqlSslMode.Disabled
+    };
+    
+    
+    public static async Task NewSqlInsertCommand(String sqlCommand)
+    {
+        await using (var conn = new MySqlConnection(builder.ConnectionString))
+        {
+            Console.WriteLine("Opening Conction");
+            await conn.OpenAsync();
+
+
+            await using (var command = conn.CreateCommand())
+            {
+                command.CommandText = sqlCommand;
+
+                // exicute statment in database 
+                await using (var reander = await command.ExecuteReaderAsync())
```

Comment:
Simply use using --> shorter

~~~csharp
 await using var command = conn.CreateCommand();
command.CommandText = sqlCommand;

// exicute statment in database 
await using var reander = await command.ExecuteReaderAsync();
~~~